src/components/hooks/useFilter.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-console */
```

Comment:
Remove all actions except setFilter and resetFilter + updateUI filter, use Wayfair example for it